### ISSUE TITLE
Accessibility features in Hyrax

### ISSUE THREAD

**Issue by bess** [samvera/samvera.github.io#101]
### Descriptive summary of request

We need a guide describing 

- the existing accessibility features in Hyrax
- the existing ways we check for accessibility
- best practices for new development to ensure we maintain and enhance these

### Existing Documentation

If you are submitting an issue related to existing documentation, include an "Existing Documentation" section
with link(s) to the document in question and describe what is incorrect. If possible, offer suggested
changes.

### Expected Document

If you were unable to find relevant documentation for your issue and are submitting a request for new documentation,
include a general summary in "Expected Document".

### Related work

If available, include a link to the software issue or PR in "Related Work".

### Tags

When possible, please tag your issue with the provided labels:

- Change to Documentation
- Documentation Request (for new documentation)
- Question (when submitting a general question)

**Comment by laritakr** [comment]
Providing for accessibility concerns is a [legal requirement](https://www.section508.gov/). I believe it was @cam156 who brought up the topic on Slack, and @awead also contributed to the discussion. This has been a concern of Penn State because they have been the subject of an accessibility related lawsuit.  As we work on this page, they should be able to provide some valuable input. 

This issue was raised on slack due to cases of accessibility features being inadvertently removed via subsequent PR's. Another concern is that newly added features may not incorporate accessibility features which already exist. Suggestions in the slack discussion included potentially including some feature specs (which are slow but may be the best way to ensure that features aren't removed), as well as adding inline documentation so the purpose of some of the code is more apparent.

[Accessibility at Penn State](http://accessibility.psu.edu/blockers/) documents some of the features they are working toward, and documents issues in Hyrax which they consider to be blockers. They have created [accessibility concern](https://github.com/samvera/hyrax/issues?q=is%3Aissue+is%3Aopen+label%3Aaccessibility-concern) and [accessibility blocker](https://github.com/samvera/hyrax/issues?q=is%3Aissue+is%3Aopen+label%3Aaccessibility-blocker) tags for documenting accessibility issues in the Hyrax backlog. 

The goals of this "best practices regarding accessibility features" page would be to: 
- [ ] List the accessibility features which have been incorporated into Hyrax
- [ ] Document a strategy for testing accessibility features
- [ ] Document methods for incorporating the supported features into new code
- [ ] Potentially get a link to this new page added into the PR template for more visibility of the issue.
